src/pages/WishAndIngredients.tsx

```diff
@@ -0,0 +1,5 @@
+const WishAndIngredients = () => {
```

Comment:
```suggestion
const WishesAndIngredients = () => {
```
Ingredients가 복수라서 Wish도 복수형으로 바꾸는 것이 좋을 것 같습니다!
(너무 소소한거라서 민망하네요....)

Reply:
절대 소소하지 않아요!! 네이밍은 아주 중요합니다ㅎㅎ 말씀해주셔서 감사합니다!! 다만, 디자이너분들이 이 페이지가 삭제되고 팝업으로 대체될 거 같다고 하셔서 페이지 자체를 지워야할 것 같네요ㅠㅠ